src/extension.ts

```diff
@@ -39,6 +39,9 @@ function runJavaServer(context: VSCode.ExtensionContext) : Thenable<StreamInfo>
 				let vmargs = getSonarLintConfiguration().get('ls.vmargs','');
 				parseVMargs(params, vmargs);
 				params.push('-jar', serverJar, '' + languageServerPort);
+				params.push(toUrl(Path.resolve(context.extensionPath, "analyzers", "sonarjs.jar")));
+				params.push(toUrl(Path.resolve(context.extensionPath, "analyzers", "sonarphp.jar")));
+				params.push(toUrl(Path.resolve(context.extensionPath, "analyzers", "sonarpython.jar")));
```

Comment:
Let say this is intentional. Another option would have been to list files in the "analyzers" folder, but I'm a bit afraid of adventurous users trying to drop random analyzers in this folder.
And since the file names doesn't contain the version, we should not have to update them.